### PR TITLE
[fix bug 1137303] Updates to Hello product page

### PR DESCRIPTION
**Pull request by alexgibson**
Still waiting on some creative approval for design tweaks, but this should be good for code review.

Also waiting on string change approval from legal.

This PR uses the same `rays-light-blue` sandstone class that is being added in #2864. Whichever merges first, I'll rebase if needed.

### CODE REVIEW

**Comment by ckprice** [comment]
Final copy for security blurb

Safe, secure, protected
Because Hello is built right into Firefox, you can rest easy knowing that your conversations and information will remain private and secure. And thanks to WebRTC-grade encryption, you’ll enjoy an added level of security. Learn more about WebRTC encryption »

**Comment by alexgibson** [comment]
Ok strings are final, design changes are approved. Removing DNM on this one :+1:

**Comment by alexgibson** [comment]
Also, @jgmize - here be our first spring campaign waffle switch: `hello-sceen-sharing`

File: bedrock/firefox/templates/firefox/hello/index.html [hunk 1]
```diff
@@ -134,26 +135,48 @@ <h3 class="heading-tertiary">{{ _('All you need is Firefox') }}</h3>
...
-          <h3 class="heading-tertiary">{{ _('Invite anyone, anywhere') }}</h3>
-
-          <p>
-            {{ _('Hello is super easy to use and your friends and family don’t even need Firefox to join your conversation.') }}
-            {{ _('As long as they have a compatible browser, they’re good to go.') }}
-          </p>
-
-          <a rel="external" href="https://support.mozilla.org/kb/which-browsers-will-work-firefox-hello-video-chat">{{ _('Learn more about compatible browsers') }}</a>
-        </li>
+        {% if l10n_has_tag('hello_screen_sharing') and waffle.switch('hello-sceen-sharing') %}
```
Comment (jgmize) [inline bedrock/firefox/templates/firefox/hello/index.html]: typo in the waffle switch name

File: bedrock/firefox/templates/firefox/hello/index.html [hunk 2]
```diff
@@ -134,26 +135,48 @@ <h3 class="heading-tertiary">{{ _('All you need is Firefox') }}</h3>
...
+            <a rel="external" href="https://support.mozilla.org/kb/which-browsers-will-work-firefox-hello-video-chat">{{ _('Learn more about compatible browsers') }}</a>
+          </li>
+        {% endif %}
         <li id="feature-secure">
           <h3 class="heading-tertiary">{{ _('Safe, secure, protected') }}</h3>
 
           <p>
             {{ _('Because Hello is built right into Firefox, you can rest easy knowing that your conversations and information will remain private and secure.') }}
-            {{ _('That’s part of our mission, and it’s behind everything we do around here.') }}
+            {% if l10n_has_tag('hello_screen_sharing') and waffle.switch('hello-sceen-sharing') %}
```
Comment (jgmize) [inline bedrock/firefox/templates/firefox/hello/index.html]: same typo :)

File: bedrock/firefox/templates/firefox/hello/index.html [hunk 3]
```diff
@@ -134,26 +135,48 @@ <h3 class="heading-tertiary">{{ _('All you need is Firefox') }}</h3>
           <p>
             {{ _('If you have the latest version of Firefox, you already have Hello.') }}
             {{ _('There’s no account or sign-in required and nothing extra to download.') }}
-            {{ _('Just start a conversation, send your friend a link and ask them to click it.') }}
-            {{ _('Simple.') }}
+            {% if l10n_has_tag('hello_screen_sharing') and waffle.switch('hello-sceen-sharing') %}
```
Comment (jgmize) [inline bedrock/firefox/templates/firefox/hello/index.html]: waffle switch typo here

File: bedrock/firefox/templates/firefox/hello/index.html (hunk 1 above)
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/hello/index.html]: heh, thanks for catching lolz

**Comment by jgmize** [comment]
I saw what I assumed was a typo in the waffle switch name, but since it's everywhere I guess I should double check: did you actually mean to use `hello-sceen-sharing` or should it be globally replaced with `hello-screen-sharing`?

**Comment by alexgibson** [comment]
It's a total typo - updating thanks @jgmize

**Comment by alexgibson** [comment]
Note: I need to rebase this now that #2831 merged. Will do this soon

File: media/css/firefox/hello/index.less [hunk 4]
```diff
@@ -141,7 +122,9 @@ main {
 }
 
 #powered-by {
-    .font-size(@smallFontSize);
+    .font-size(@baseFontSize);
+    .open-sans-italic;
```
Comment (craigcook) [inline media/css/firefox/hello/index.less]: I saw this mixin was added recently but I'm not sure it's needed. Is this better than just <code>font-style: italic</code>? The mixin explicitly declares the font-family but as far as I can tell that's already inherited.

**Comment by craigcook** [comment]
Looks good, just a minor nitpick on what might be a redundant mixin. r+wc :rugby_football:

File: media/css/firefox/hello/index.less (hunk 4 above)
Comment (alexgibson) [inline media/css/firefox/hello/index.less]: In this case it does indeed seem to be redundant, removing! :-1:

**Comment by alexgibson** [comment]
Thanks @craigcook updated!

**Pull request merged**